client/blocks/product-purchase-features-list/customize-theme.jsx

```diff
@@ -1,24 +1,39 @@
 import { localize } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
 import customizeImage from 'calypso/assets/images/illustrations/dashboard.svg';
+import QueryActiveTheme from 'calypso/components/data/query-active-theme';
 import PurchaseDetail from 'calypso/components/purchase-detail';
+import withBlockEditorSettings from 'calypso/data/block-editor/with-block-editor-settings';
+import getCustomizeUrl from 'calypso/state/selectors/get-customize-url';
+import { getActiveTheme } from 'calypso/state/themes/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
-function getCustomizeLink( selectedSite ) {
-	return '/customize/' + selectedSite.slug;
-}
+const CustomizeTheme = ( { translate, blockEditorSettings, areBlockEditorSettingsLoading } ) => {
+	const isFSEActive = blockEditorSettings?.is_fse_active;
+	const siteId = useSelector( getSelectedSiteId );
+	const currentThemeId = useSelector( getActiveTheme );
+	const customizeLink = useSelector( ( state ) =>
+		getCustomizeUrl( state, currentThemeId, siteId, isFSEActive )
+	);
 
-export default localize( ( { selectedSite, translate } ) => {
 	return (
 		<div className="product-purchase-features-list__item">
+			<QueryActiveTheme siteId={ siteId } />
 			<PurchaseDetail
+				isSubmitting={ areBlockEditorSettingsLoading }
 				icon={ <img alt="" src={ customizeImage } /> }
 				title={ translate( 'Advanced customization' ) }
 				description={ translate(
 					"Change your site's appearance in a few clicks, with an expanded " +
 						'selection of fonts and colors.'
 				) }
-				buttonText={ translate( 'Start customizing' ) }
-				href={ getCustomizeLink( selectedSite ) }
+				buttonText={
+					areBlockEditorSettingsLoading ? translate( 'Loading…' ) : translate( 'Start customizing' )
```

Comment:
While reviewing this I noticed that most of the time when linking to the Site editor, the text is "Edit site". What do you think of doing the same here?
(Change to "Edit site" for when linking to the Site editor and leave "Start customizing" when going to the Customizer)

Reply:
I think we should change the text to 'Edit site' when navigating to the full site editor and 'Start customizing' when navigating to customizer. As a developer this will be more consistent with other text related to FSE (for example, the modal popup after activating a theme) and it will also help make the connection between FSE and 'Edit Site' text for customers as well.